packages/flame/README.md

```
@@ -0,0 +1,481 @@
+- Buttons/controls are keyboard-activatable (Enter/Space)
+
+Editor integration
+
+- Editor selector is only rendered when `openInEditorUrl` is provided
+- Selected editor persists and is sent to the server opener
+
+## Quick start (HTTP server)
+
+Use the built-in displayer to render the error page and respond to the request. Optionally add an endpoint to open files in your editor.
+
+```ts
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)